lab-kyle/test/note-route-test.js

```diff
@@ -0,0 +1,164 @@
+'use strict';
```

Comment:
These look great so far, the delete tests still need to be completed as you're aware.  Keep up the good work!